ParabolInc/parabol
# Jira server pagination

# Description

Fixes #6605

## Testing scenarios

- Go to the scope phase of a sprint poker meeting
- Open Jira Server tab
- Scroll till the end and see that new issues are loaded
